cmd/agent/app/integrations.go

```diff
@@ -192,23 +193,27 @@ func tuf(args []string) error {
 	}
 	args = append(args, implicitFlags...)
 
-	// Proxy support
-	proxies := config.GetProxies()
-	if proxies != nil {
-		proxyFlags := fmt.Sprintf("--proxy=%s", proxies.HTTPS)
```

Comment:
this is a [supported `pip` flag](https://pip.pypa.io/en/stable/user_guide/#using-a-proxy-server), why doesn't our custom pip (with tuf+in-toto enabled) support it?

cc @trishankatdatadog

Reply:
I agree: our pip should "pass" the `--proxy` flag to `tuf` which would, in turn, pass it to `requests`. I'll make a note of this for a future feature, but for now, I would recommend using the `HTTP(S)_PROXY` environment variables, please. Thanks!